s/toolbox/hamster-html/hamster.test.ts

```diff
@@ -0,0 +1,28 @@
+
+import { html, render } from "./html.js";
+import { Suite, assert, expect } from "cynic";
+
+export default <Suite>{
+	"santization": {
+		"sanitization occurs by default": async () => {
+			const input = html`${"<div></div>"}`
+			const output = "&lt;div&gt;&lt;/div&gt;"
+
+			assert(render(input)===output,"sanitization does not occur")
```

Comment:
> "sanitization does not occur"

is this correct? i think it should say "sanitization does occur"

Reply:
I think it is... Since the goal is to test if sanitization occurs, then the error message should say otherwise, right?

Reply:
i think the assert is worded wrong --
the assert is checking that the sanitization *does* occur,
but says "does not"

?

Reply:
i think the assert is worded wrong --
the assert is checking that the sanitization does occur,
but says "does not"

?

Reply:
> // benefits of 'assert'
      //  - you get a stack trace
      //  - you can provide a custom message for each failure

here is how I think about it: 
- the purpose of the test is to check if sanitization occurs
- if there is a failure, then it means sanitization doesn't occur
so that is why I worded the custom message for the failure like that

Reply:
oh hahah, okay, that actually makes sense. it sure is confusing :laughing: 

maybe a clear way to express it might be `sanitization should occur, but doesn't`

Reply:
okay 👍